README.md

```diff
@@ -1,10 +1,81 @@
 # Curriculum
 This project provides study material for [Telegram channel](https://t.me/learniteasy2)
 
-## Draft overview
-* Result of the course: make a working TODO list application.
-  * Success is defined by employed students  
-* Topics:
-  * basics
-  * git basics
-  * 
+## Goal
+Success is defined by employed students
+
+## Plan
+🌟 Season 1: Foundations of IT
+
+	•	Week 1: Introduction to Computers
+	•	Understanding hardware vs. software
+	•	How computers work: CPU, memory, storage
+	•	Week 2: Operating Systems Basics
+	•	Navigating Windows, macOS, and Linux
+	•	Command-line introduction
+	•	Week 3: Networking Fundamentals
+	•	What is the Internet?
+	•	Basic networking concepts and terminology
+	•	Week 4: Cybersecurity Essentials
+	•	Understanding threats and protection
+	•	Best practices for staying safe online
+
+🌟 Season 2: Programming Fundamentals
+
+	•	Week 1: Introduction to Programming Concepts
+	•	Variables, data types, and operators
+	•	Control structures: loops and conditionals
+	•	Week 2: Getting Started with Python
+	•	Setting up your development environment
+	•	Writing and running your first JavaScript script
+	•	Week 3: Functions and Modules
+	•	Creating reusable code blocks
+	•	Importing and using modules
+	•	Week 4: Working with Data
+	•	Lists, dictionaries, and data manipulation
+	•	Basic file I/O operations
+
+🌟 Season 3: Web Development Basics
+
+	•	Week 1: HTML - Structuring the Web
+	•	Building your first webpage
+	•	Understanding tags and elements
+	•	Week 2: CSS - Styling the Web
+	•	Adding styles to your webpage
+	•	Layouts, colors, and fonts
+	•	Week 3: JavaScript Basics
+	•	Making your webpage interactive
+	•	Basic syntax and DOM manipulation
+	•	Week 4: Building a Simple Website (TODO list)
+	•	Combining HTML, CSS, and JavaScript
+	•	Project: Your personal portfolio site
+
+🌟 Season 4: Exploring Specializations
+
+	•	Week 1: Introduction to Databases
```

Comment:
Maybe we could put it at the end of season two? This would allow students to complete the TODO list in season three. I would also add a topic about APIs and maybe how to use postman.

Reply:
Good point about APIs and databases. In season 3, we can give a very brief intro into these topics. However, in extra section we might give a broader view (sql, no-sql, how to choose and etc).
Postman is redundant for this course. Let's not dive into tools, we can list/recommend tools which they can use or learn, but this is up to them.